api/src/Controller/Stores/DeleteProduct.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller\Stores;
+
+use App\Controller\BaseController;
+
+use Atlas\Query\Delete;
+
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class DeleteProduct extends BaseProduct
+{
+
+
+    public function buildResource(Request $request, Response $response, $params): array
+    {
+        if (!$_SESSION['IS_ADMIN']) {
+            return [
+            BaseController::RESULT_TYPE,
+            $this->errorResponse($request, $response, 'Permission Denied', 'Permission Denied', 403)];
+        }
+
+        $error = $this->getProduct($request, $response, $product, $params);
```

Comment:
Feels a git complicated to just verify there is a row to delete.

The way (not Atlas) I have been doing it now is
```
        $sql = "DELETE FROM `Registrations` WHERE RegistrationID = $id";
        $sth = $this->container->db->prepare($sql);
        $sth->execute();
        if ($sth->rowCount() == 0) {
```

Reply:
The row-count trick would also work here, I believe, so that's possible.